src/FlashList.tsx

```diff
@@ -69,7 +70,10 @@ class FlashList<T> extends React.PureComponent<
   private rlvRef?: RecyclerListView<RecyclerListViewProps, any>;
   private stickyContentContainerRef?: PureComponentWrapper;
   private listFixedDimensionSize = 0;
-  private transformStyle = { transform: [{ scaleY: -1 }] };
+  private transformStyle =
```

Comment:
We keep our platform-specific code inside the native/ directory. Can you please move it there?
This also doesn't look like an official spec. Does FlatList also do this internally or is this optimization only part of a certain patch?